src/BaseComponent.php

```diff
@@ -160,6 +179,19 @@ public function getInputState(): array
         return $this->inputState;
     }
 
+    /** @return mixed */
+    public function execute()
+    {
+        if (!$this->isSyncAction()) {
+            $this->run();
+        }
+
+        $action = $this->getConfig()->getAction();
```

Comment:
to snad patri do else, to funguje jen nahodou

Reply:
OMFG díky